ui/desktop/app/components/proxy-url/index.hbs

```diff
@@ -0,0 +1,34 @@
+<div class='proxy-url-container'>
```

Comment:
Non-blocker- Is this a generic component that can be used in other screens or a resource-specific one? 

If it is a resource-specific one, we should consider structuring it something like `session/proxy-url` because right now all the components within `app/components` seem generic and used all over the desktop app

Reply:
Yeah I can make that change.